src/decode.rs

```diff
@@ -790,14 +791,20 @@ unsafe fn read_pal_indices(
             pal_tmp[offset..][..len].fill(filler);
         }
     }
-    (pal_dsp.pal_idx_finish)(
-        pal_idx.unwrap_or(pal_tmp).as_mut_ptr(),
-        pal_tmp.as_ptr(),
-        bw4 as c_int * 4,
-        bh4 as c_int * 4,
-        w4 as c_int * 4,
-        h4 as c_int * 4,
-    );
+
+    // SAFETY: Unsafe asm call, checkasm tests have verified that the signature is
```

Comment:
Is that all `checkasm` does?

Reply:
Naw, looks like the checkasm tests are full unit tests for the asm and C fallback routines, but since we're not running the checkasm tests against Rust the only thing it's verified for us is that the function signatures are correct. I'm open to better ways of wording this if you have suggestions.

Reply:
If we're not running those tests, signature checking isn't even happening either right?  Can we just say eventually will be checked by checkasm?

Reply:
Eh, I don't think it makes sense to say that will eventually be checked, but I can reword it to remove the reference to checkasm.